app/components/side-nav/product-switcher/index.ts

```diff
@@ -0,0 +1,87 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import type IntlService from 'ember-intl/services/intl';
+
+import styles from './index.scss';
+import type { SwitcherMenuItem } from '..';
+import type ConfigurationService from 'irene/services/configuration';
+
+export interface SideNavProductSwitcherSignature {
+  Args: {
+    isCollapsed: boolean;
+    productSwitcherFilterKey: string;
+    classes: {
+      menuItemText?: string;
+      menuItemLink?: string;
+      menuItemLinkActive?: string;
+      menuItemTooltip?: string;
+      menuItemIcon?: string;
+    };
+  };
+  Element: HTMLElement;
+}
+
+export default class SideNavProductSwitcherComponent extends Component<SideNavProductSwitcherSignature> {
+  @service declare intl: IntlService;
+  @service declare configuration: ConfigurationService;
+
+  @tracked anchorRef: HTMLElement | null = null;
+
+  get classes() {
+    return {
+      switcherModalArrow: styles['switcher-popover-arrow'],
+    };
+  }
+
+  get orgIsAnEnterprise() {
+    return this.configuration.serverData.enterprise;
+  }
+
+  get isSidebarExpanded() {
+    return !this.args.isCollapsed;
+  }
+
+  get switcherMenuItems() {
+    const allMenuItems: SwitcherMenuItem[] = [
+      {
+        id: 'vp-svg',
+        svg: 'ak-svg/vp-indicator',
+        label: this.orgIsAnEnterprise
+          ? this.intl.t('vapt')
+          : this.intl.t('appknox'),
+        route: 'authenticated.dashboard.projects',
+        key: 'storeknox',
+      },
+      {
+        id: 'sm-svg',
+        svg: 'ak-svg/sm-indicator',
+        label: this.orgIsAnEnterprise
+          ? this.intl.t('appMonitoring')
+          : this.intl.t('storeknox'),
+        route: 'authenticated.storeknox.discover',
+        key: 'appknox',
+      },
+    ];
+
+    return allMenuItems.filter(
+      (item) => item.key === this.args.productSwitcherFilterKey
```

Comment:
this should be a not comparison Eg: if active product is `vapt` then everything except `vapt`